CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ NOTE:
 
 ## Table of contents
 
-- [v1.1.0-alpha.1](#v110---alpha1)
+- [v1.1.0-rc.1](#v110---rc1)
```

Comment:
Please put here the release number, which is r2.1, as defined in https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API_Release_Guidelines.md:

> After a meta-release of an API through release rx.y, the next release number for this API is rx+1.1 (y resets to 1)

Reply:
Thanks, done.